ccm-connector/src/main/java/com/sequenceiq/cloudbreak/ccmimpl/termination/DefaultCcmResourceTerminationListener.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.cloudbreak.ccmimpl.termination;
+
+import java.util.UUID;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.google.common.base.Throwables;
+import com.sequenceiq.cloudbreak.ccm.termination.CcmResourceTerminationListener;
+import com.sequenceiq.cloudbreak.ccmimpl.altus.GrpcMinaSshdManagementClient;
+
+/**
+ * Default CCM resource termination listener.
+ */
+@Component
+public class DefaultCcmResourceTerminationListener implements CcmResourceTerminationListener {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultCcmResourceTerminationListener.class);
+
+    @Inject
+    private GrpcMinaSshdManagementClient grpcMinaSshdManagementClient;
+
+    @Override
+    public void deregisterCcmSshTunnelingKey(@Nonnull String actorCrn, @Nonnull String accountId, @Nonnull String keyId) {
+        if (grpcMinaSshdManagementClient != null) {
```

Comment:
I don't think we need to null-check the `grpdMinaSshdManagementClient`. The service will fail to start if the bean can't be wired in.

Reply:
Fixed.